### PR TITLE
Remove inactive merchants (websites long down)

### PR DESCRIPTION
**Pull request by qertoip**

### CODE REVIEW

File: _data/merchants.yml [hunk 1]
```diff
@@ -118,24 +118,18 @@
...
-    - name: algoStrategic - Internet Marketing and Web Development
-      url: https://algostrategic.com
     - name: Emmanuel Galang, Canada Immigration and Refugee Lawyer
       url: https://galanglaw.com/
     - name: Web Developer - Python with Django web framework
       url: http://www.voteforrodneylewis.com
     - name: FlokiNET
       url: https://flokinet.is
-    - name: Nerdzy Lawn Care
-      url: http://www.nerdzy.net
```
Comment (mattcode55) [inline _data/merchants.yml]: Looks like they're still up, still doing lawn care and still accepting Monero for payment.

http://www.nerdzy.net/index.php/lawn-care/

>Methods of payment: Our team believes in innovation and that is why we accept a variety of currencies at Nerdzy. As a whole, we accept: cash (USD), PayPal/Credit, Dogecoin (DOGE), Bitcoin (BTC), Monero/Bitmonero (XMR), and Ethereum (ETH)

File: _data/merchants.yml [hunk 2]
```diff
@@ -118,24 +118,18 @@
       url: http://www.guitartheoryrevolution.info/blog/guitar-theory-revolution-store/
     - name: MyMonero Web-based Wallet
       url: https://mymonero.com
-    - name: Pradeep Atluri, Psychiatrist, New York
-      url: http://dr.mindsci.com/
```
Comment (mattcode55) [inline _data/merchants.yml]: +1, is down

File: _data/merchants.yml [hunk 3]
```diff
@@ -118,24 +118,18 @@
...
     - name: Simple, no non-sense hosting
       url: https://rootbox.host/
     - name: Web Developer - Stefanos
       url: http://www.stefanosioannou.com/web-development-monero-accepted
     - name: XMR.to Monero to Bitcoin Payment Service
       url: https://xmr.to
-    - name: algoStrategic - Internet Marketing and Web Development
-      url: https://algostrategic.com
```
Comment (mattcode55) [inline _data/merchants.yml]: +1, is down

**Comment by qertoip** [comment]
Thanks again, @mattcode55! I added nerdzy.net back to the merchants. It was down at the time of my removal but apparently they fixed their website.

File: _data/merchants.yml [hunk 4]
```diff
@@ -39,7 +39,9 @@
     - name: Magnetic Exchange
       url: https://magneticexchange.com
     - name: ArbitrCoin
-      url: https://arbitrcoin.com      
+      url: https://arbitrcoin.com
+    - name: Nerdzy Lawn Care
+      url: http://www.nerdzy.net
```
Comment (mattcode55) [inline _data/merchants.yml]: I think that you've added Nerdzy as an exchange by a mistake.

**Comment by mattcode55** [comment]
@nerdzy Do you accept Monero on other parts of your website, or just for lawn care?

File: _data/merchants.yml (hunk 1 above)
Comment (Nerdzy) [inline _data/merchants.yml]: Yes, we're still up, we moved the page to a different location on the site, however, it's still there.

File: _data/merchants.yml (hunk 3 above)
Comment (Nerdzy) [inline _data/merchants.yml]: Also, our site was down for a short while today, but it's back online after resolving a network IP address change.

**Comment by qertoip** [comment]
@mattcode55 I moved nerdzy.net back to the correct section and squashed the branch into a single cleaning commit. Thanks so much!

**Comment by qertoip** [comment]
EOF newline removed as requested by @MaxXor.

**Pull request merged**